### PR TITLE
Create screenshots with ElementBot

### PR DESCRIPTION
**Pull request by jmartinesp**
This change triggers the usual CI workflows automatically when the record screenshots workflow is done.

### CODE REVIEW

File: libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/Color.kt [hunk 1]
```diff
@@ -25,7 +25,7 @@ val LightGrey = Color(0x993C3C43)
 @ShowkaseColor(name = "DarkGrey", group = "Material Design")
 val DarkGrey = Color(0x99EBEBF5)
 
-val AvatarGradientStart = Color(0xFF4CA1AF)
+val AvatarGradientStart = Color.Red
```
Comment (bmarty) [inline libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/Color.kt:28]: Do not forget to revert this change :)

**Review by bmarty — COMMENTED** [review]
OK, I think this is working as expected, thanks for the changes!
Will approve the PR after the clean up :)

File: libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/Color.kt (hunk 1 above)
Comment (jmartinesp) [reply libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/Color.kt:28]: I'll do 1 more change to try to simplify the script a bit more and then revert this change if it works 👍

Comment (jmartinesp) [reply libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/Color.kt]: Done!

**Review by bmarty — APPROVED** [review]
LGTM, thanks for having fixed this pain in the neck!
